### DISCUSSION TITLE
Count occurrences of value in data

### DISCUSSION THREAD

**Discussion by nabelekt** [grafana/grafana#64460] [Dashboards, Panels & Visualizations]
I am looking for a Transform that can count occurrences of each unique value in data. This request was previously described here: https://github.com/grafana/grafana/issues/25594#issuecomment-643993616. But that was added after [#25594 ](https://github.com/grafana/grafana/issues/25594) was closed. As far as I can tell, the feature has not yet been added.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82834. Feel free to continue the discussion around this there. Thank you!
